Resources/Prototypes/Modifications/Drug/reagents.yml

```
@@ -0,0 +1,52 @@
+- type: entity
+  name: flash dose
+  parent: Pill
+  id: PillFlashDose
+  components:
+  - type: SolutionContainerManager
+    solutions:
+      food:
+        maxVol: 20
+        reagents:
+        - ReagentId: FlashDose
+          Quantity: 15
```

yamllint

[error] no new line character at the end of file

(new-line-at-end-of-file)